packages/console/src/pages/SignInExperience/components/SignInMethodsForm.tsx

```diff
@@ -26,6 +27,16 @@ const SignInMethodsForm = () => {
 
   useEffect(() => {
     if (primaryMethod) {
+      if (oldPrimaryMethod.current) {
+        // The secondary sign in method should select the old primary method by default.
+        setValue(`signInMethods.${oldPrimaryMethod.current}`, true);
```

Comment:
Can we do this in the `onChange` handler of the primary select field at line 92?
This way you can have access to both old and new value, and don't have to introduce this `oldPrimaryMethod` ref

Reply:
Good call. Updated.